docs/tutorials/11_working_with_classical_optimization_solvers.ipynb

```diff
@@ -0,0 +1,578 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Working with Classical Optimization Solvers"
```

Comment:
I was wondering if there was a "better" title that `Working with Classical Optimization Solvers` I was thinking about something like `Using Classical Optimization Solvers and Models with Qiskit` or `Interoperating with Classical Optimization Solvers and Models` - something that perhaps in the title makes it feel more like they are being talked about/used in the context of Qiskit Optimization. I added Model there too since we use the representation and I think can do so independent of the solvers.

Reply:
OK. I renamed "Using Classical Optimization Solvers and Models with Qiskit Optimization".